### PR TITLE
add styles to header

### PR DESCRIPTION
**Pull request by kovpaka**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -30,31 +37,120 @@
...
+    margin-left: 35%;
+    padding: 3px;
+    content: '';
+    -webkit-transform: rotate(45deg);
+    transform: rotate(45deg);
+    border: solid var(--main-elements-color);
+    border-width: 0 2px 2px 0;
+}
+
+.navbar__link:hover::before {
```
Comment (siarheikarabko) [inline style.css:67]: I don't think it makes sense to use hover on mobile (touch) devices. Probably `focus` will be better.

File: style.css [hunk 2]
```diff
@@ -30,31 +37,120 @@
...
 
 .banner {
     min-height: 520px;
-    padding: 115px 35% 50px;
+    padding: 115px 20% 50px;
     text-align: center;
-    background-color: rgb(245, 132, 98);
+    letter-spacing: 2.5px;
+    background-image: url(../images/banner__background.jpg);
+    background-position: 100% 4%;
```
Comment (siarheikarabko) [inline style.css:116]: Please, try to use
```
    background-size: cover;
    background-position: top;
```

File: style.css [hunk 3]
```diff
@@ -30,31 +37,120 @@
...
     overflow-wrap: break-word;
 }
 
 .banner__main-text {
     font-family: 'PT Sans', sans-serif;
     font-size: 40px;
     font-weight: 700;
+    line-height: 1.4em;
+    display: inline;
+    padding-bottom: 3px;
```
Comment (siarheikarabko) [inline style.css:126]: Do we really need it?

File: style.css [hunk 4]
```diff
@@ -30,31 +37,120 @@
...
+    margin-bottom: 23px;
+    padding: 10px;
+}
+
+.navbar__link--active {
+    color: var(--focus-elements-color);
+}
+
+.navbar__link::before {
+    display: inline-block;
```
Comment (siarheikarabko) [inline style.css:55]: Let's discuss this element, I'd recommend using absolute positioning.

File: index.html [hunk 5]
```diff
@@ -20,15 +20,50 @@
...
+                    </li>
+                    <li class="navbar__link">
+                        <a href="###">Contact</a>
+                    </li>
+                </ul>
+
+                <div class="navbar__burger">
+                    <img class="navbar__burger-ico" src="./images/ico-bar.svg" alt="menu">
+
+                    <div class="navbar__dropdown-content">
```
Comment (siarheikarabko) [inline index.html:51]: I know that it's optional but anyway, why don't you use `<ul>` here as well?)

File: style.css [hunk 6]
```diff
@@ -30,31 +37,120 @@
...
 }
 
 .banner {
     min-height: 520px;
-    padding: 115px 35% 50px;
+    padding: 115px 20% 50px;
     text-align: center;
-    background-color: rgb(245, 132, 98);
+    letter-spacing: 2.5px;
+    background-image: url(../images/banner__background.jpg);
```
Comment (siarheikarabko) [inline style.css:115]: Please, check the path.

File: index.html [hunk 7]
```diff
@@ -20,15 +20,50 @@
...
             <nav class="navbar">
-                <div class="navbar__logo">Site name</div>
-                <div class="navbar__links">Links</div>
+                <div class="navbar__logo">
+                    <a href="index.html" class="navbar__logo-link">Site Name</a>
+                </div>
+
+                <ul class="navbar__links">
+                    <li class="navbar__link">
+                        <a href="###" class="navbar__link--active">Home</a>
```
Comment (siarheikarabko) [inline index.html:29]: Please, review the link class name

File: style.css [hunk 8]
```diff
@@ -1,23 +1,30 @@
+:root {
+    --main-elements-color: #637f83;
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply style.css:2]: got it

File: style.css [hunk 9]
```diff
@@ -1,23 +1,30 @@
...
 ::after,
 ::before {
     font-family: 'Source Sans Pro', sans-serif !important;
     font-size: 14px;
     font-weight: 400;
     box-sizing: border-box;
-    color: #637f83;
+    text-decoration: none;
+    color: var(--main-elements-color);
+    outline: none !important;
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply style.css:16]: Got it.

File: style.css [hunk 10]
```diff
@@ -30,31 +37,120 @@
...
+.navbar__burger-ico {
+    fill: var(--main-elements-color);
+}
+
+.navbar__dropdown-content {
+    position: absolute;
+    z-index: 1;
+    display: none;
+    min-width: 160px;
+    background-color: white;
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply style.css:89]: Got it. Fixed;

File: style.css [hunk 11]
```diff
@@ -30,31 +37,120 @@
 .navbar__links {
     font-size: 16px;
     font-weight: 700;
+    text-transform: uppercase;
+}
+
+.navbar__link {
+    float: left;
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply style.css:44]: Done.

File: style.css [hunk 12]
```diff
@@ -30,31 +37,120 @@
...
+    margin-bottom: 10px;
+    margin-bottom: 23px;
+    padding: 10px;
+}
+
+.navbar__link--active {
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply style.css:50]: -

File: style.css (hunk 1 above)
Comment (kovpaka) [reply style.css:67]: Got it

File: style.css (hunk 2 above)
Comment (kovpaka) [reply style.css:116]: It's really helped me, thank you.

File: style.css (hunk 3 above)
Comment (kovpaka) [reply style.css:126]: Yep.

File: style.css (hunk 4 above)
Comment (kovpaka) [reply style.css:55]: Fixed.

File: index.html (hunk 5 above)
Comment (kovpaka) [reply index.html:51]: There are no these elements anymore.

File: style.css (hunk 6 above)
Comment (kovpaka) [reply style.css:115]: Done.

File: index.html (hunk 7 above)
Comment (kovpaka) [reply index.html:29]: Done

**Pull request merged**